gradle/dependencies.gradle

```diff
@@ -15,17 +15,23 @@
  */
 
 def versions = [
-    kotlin    : "1.2.60",
-    support   : "27.1.0"
+    kotlin    : "1.3.10"
+]
+
+def androidx = [
+    annotations      : "androidx.annotation:annotation:1.0.0",
+    appcompat        : "androidx.appcompat:appcompat:1.0.0",
+    constraintLayout : "androidx.constraintlayout:constraintlayout:1.1.2",
+    v4               : "androidx.legacy:legacy-support-v4:1.0.0"
```

Comment:
Worth trying to split this up while we're at it?

Reply:
ha, ya, i was thinking the same. will take a look since you pointed it out

Reply:
was actually unused, lulz